frontend/src/components/pages/RegistrantExperience/RegistrationFooter.tsx

```diff
@@ -34,6 +34,7 @@ const RegistrationFooter = ({
       padding="20px"
```

Comment:
I think the designs say that the button on the review cart page should be "Go to Checkout" not "pay". Could you fix that up in this PR? On the review info step (what tejas is working on, also part of step 4) the CTA is next, but when it comes to the cart review page, the CTA is "go to checkout"